sections/Deals/AcceptOrReject/AcceptOrRejectBox/AcceptOrRejectBox.tsx

```diff
@@ -0,0 +1,298 @@
+import Image from 'next/image';
+import styled from 'styled-components';
+import { FC, useEffect, useState, useMemo } from 'react';
+
+import Info from 'assets/svg/info.svg';
+
+import Connector from 'containers/Connector';
+import TransactionData from 'containers/TransactionData';
+
+import { Status } from 'components/DealStatus';
+import QuestionMark from 'components/QuestionMark';
+import { FlexDivRowCentered, Tooltip } from 'components/common';
+import ConfirmTransactionModal from 'components/ConfirmTransactionModal';
+
+import {
+	Container,
+	ContentContainer,
+	ActionBoxInputLabel,
+	InputContainer,
+	ActionBoxInput,
+	ActionBoxMax,
+	ActionBoxHeaderWrapper,
+	ActionBoxHeader,
+	FlexDivCenterRow,
+	ActionButton,
+} from 'sections/shared/common';
+
+import { GasLimitEstimate } from 'constants/networks';
+import { statusToText, swimmingPoolID } from 'constants/pool';
+import { TransactionType } from 'constants/transactions';
+
+import AcceptOrRejectError from '../AcceptOrRejectError';
+
+interface AcceptOrRejectDealBoxProps {
+	poolId?: string;
+	onSubmit: () => void;
+	inputValue: number | string;
+	setInputValue: (val: number | string) => void;
+	setIsMaxValue: (val: boolean) => void;
+	userPoolBalance: string | null;
+	gasLimitEstimate: GasLimitEstimate;
+	purchaseCurrency: string | null;
+	dealRedemptionData: {
+		status: Status;
+		maxProRata: number;
+		isOpenEligible: boolean;
+		purchaseTokenTotalForDeal: number;
+		totalAmountAccepted: number;
+	};
+}
+
+const AcceptOrRejectDealBox: FC<AcceptOrRejectDealBoxProps> = ({
+	poolId,
+	onSubmit,
+	inputValue,
+	setInputValue,
+	setIsMaxValue,
+	userPoolBalance,
+	gasLimitEstimate,
+	purchaseCurrency,
+	dealRedemptionData,
+}: AcceptOrRejectDealBoxProps) => {
+	const { walletAddress } = Connector.useContainer();
+	const { setGasPrice, txType, setTxType } = TransactionData.useContainer();
+
+	const [showTooltip, setShowTooltip] = useState(false);
+	const [showTxModal, setShowTxModal] = useState(false);
+	const [isDealAccept, setIsDealAccept] = useState(true);
+
+	const isWithdraw = !isDealAccept;
+	const isEmptyInput = inputValue === '' || Number(inputValue) === 0;
+	const hasAmount = Number(isEmptyInput ? 0 : inputValue) > 0;
+	const isPoolDisabled = [swimmingPoolID].includes(poolId ?? '');
```

Comment:
purchasing for this pool is over so we can remove this filter now. we can hide the withdrawn amount on the UI though. I do not think it will display properly. we should disable the create deal section for this pool as well.

Reply:
purchasing for this pool is over so we can remove the filter now. we should disable the create deal section from showing for this pool id. people can withdraw now if they want

Reply:
Done